### PR TITLE
feat: add Name field to StackParameter

### PR DESCRIPTION
**Pull request by radnov**

### CODE REVIEW

File: pkg/stack/stack.go [hunk 1]
```diff
@@ -140,15 +140,15 @@ var DHIS2DB = model.Stack{
...
-		"DATABASE_ID":               {Priority: 1},
-		"DATABASE_SIZE":             {Priority: 2, DefaultValue: &dhis2DBDefaults.dbSize},
-		"DATABASE_NAME":             {Priority: 3, DefaultValue: &dhis2DBDefaults.dbName},
-		"DATABASE_PASSWORD":         {Priority: 4, DefaultValue: &dhis2DBDefaults.dbPassword},
-		"DATABASE_USERNAME":         {Priority: 5, DefaultValue: &dhis2DBDefaults.dbUsername},
-		"DATABASE_VERSION":          {Priority: 6, DefaultValue: &dhis2DBDefaults.dbVersion},
-		"RESOURCES_REQUESTS_CPU":    {Priority: 7, DefaultValue: &dhis2DBDefaults.resourcesRequestsCPU},
-		"RESOURCES_REQUESTS_MEMORY": {Priority: 8, DefaultValue: &dhis2DBDefaults.resourcesRequestsMemory},
-		"CHART_VERSION":             {Priority: 9, DefaultValue: &dhis2DBDefaults.chartVersion},
+		"DATABASE_ID":               {Priority: 1, Name: "Database ID"},
```
Comment (tonsV2) [inline pkg/stack/stack.go:143]: Maybe just "Database"?

File: pkg/stack/stack.go [hunk 2]
```diff
@@ -176,28 +176,29 @@ var dhis2DBDefaults = struct {
 	resourcesRequestsMemory: "256Mi",
 }
 
+// TODO Use DHIS2Core stack in DHIS2 stack to eliminate having to list same parameters twice
```
Comment (tonsV2) [inline pkg/stack/stack.go:179]: Do we still want this comment?

File: pkg/stack/stack.go [hunk 3]
```diff
@@ -176,28 +176,29 @@ var dhis2DBDefaults = struct {
...
-		"JAVA_OPTS":                       {Priority: 13, DefaultValue: &dhis2CoreDefaults.javaOpts},
-		"CHART_VERSION":                   {Priority: 14, DefaultValue: &dhis2CoreDefaults.chartVersion},
-		"DATABASE_HOSTNAME":               {Priority: 0, Consumed: true},
-		"DATABASE_NAME":                   {Priority: 0, Consumed: true},
-		"DATABASE_PASSWORD":               {Priority: 0, Consumed: true},
-		"DATABASE_USERNAME":               {Priority: 0, Consumed: true},
+		"IMAGE_TAG":                       {Priority: 1, Name: "Image Tag", DefaultValue: &dhis2CoreDefaults.imageTag},
+		"IMAGE_REPOSITORY":                {Priority: 2, Name: "Image Repository", DefaultValue: &dhis2CoreDefaults.imageRepository},
+		"IMAGE_PULL_POLICY":               {Priority: 3, Name: "Image Pull Policy", DefaultValue: &dhis2CoreDefaults.imagePullPolicy, Validator: imagePullPolicy},
+		"DHIS2_HOME":                      {Priority: 4, Name: "DHIS2 Home", DefaultValue: &dhis2CoreDefaults.dhis2Home},
```
Comment (tonsV2) [inline pkg/stack/stack.go:187]: "DHIS2 Home folder"?

File: pkg/stack/stack.go [hunk 4]
```diff
@@ -242,29 +243,29 @@ var DHIS2 = model.Stack{
...
-		"LIVENESS_PROBE_TIMEOUT_SECONDS":  {Priority: 18, DefaultValue: &dhis2CoreDefaults.livenessProbeTimeoutSeconds},
-		"READINESS_PROBE_TIMEOUT_SECONDS": {Priority: 19, DefaultValue: &dhis2CoreDefaults.readinessProbeTimeoutSeconds},
-		"STARTUP_PROBE_FAILURE_THRESHOLD": {Priority: 20, DefaultValue: &dhis2CoreDefaults.startupProbeFailureThreshold},
-		"STARTUP_PROBE_PERIOD_SECONDS":    {Priority: 21, DefaultValue: &dhis2CoreDefaults.startupProbePeriodSeconds},
-		"JAVA_OPTS":                       {Priority: 22, DefaultValue: &dhis2CoreDefaults.javaOpts},
-		"CHART_VERSION":                   {Priority: 23, DefaultValue: &dhis2CoreDefaults.chartVersion},
+		"IMAGE_TAG":                       {Priority: 1, Name: "Image Tag", DefaultValue: &dhis2CoreDefaults.imageTag},
+		"IMAGE_REPOSITORY":                {Priority: 2, Name: "Image Repository", DefaultValue: &dhis2CoreDefaults.imageRepository},
+		"IMAGE_PULL_POLICY":               {Priority: 3, Name: "Image Pull Policy", DefaultValue: &dhis2CoreDefaults.imagePullPolicy, Validator: imagePullPolicy},
+		"DATABASE_ID":                     {Priority: 4, Name: "Database ID"},
```
Comment (tonsV2) [inline pkg/stack/stack.go:249]: Strip the id part?

File: pkg/stack/stack.go [hunk 5]
```diff
@@ -242,29 +243,29 @@ var DHIS2 = model.Stack{
...
+		"DATABASE_NAME":                   {Priority: 5, Name: "Database Name", DefaultValue: &dhis2DBDefaults.dbName},
+		"DATABASE_PASSWORD":               {Priority: 6, Name: "Database Password", DefaultValue: &dhis2DBDefaults.dbPassword},
+		"DATABASE_SIZE":                   {Priority: 7, Name: "Database Size", DefaultValue: &dhis2DBDefaults.dbSize},
+		"DATABASE_USERNAME":               {Priority: 8, Name: "Database Username", DefaultValue: &dhis2DBDefaults.dbUsername},
+		"DATABASE_VERSION":                {Priority: 9, Name: "Database Version", DefaultValue: &dhis2DBDefaults.dbVersion},
+		"INSTALL_REDIS":                   {Priority: 10, Name: "Install Redis", DefaultValue: &dhis2Defaults.installRedis},
+		"DHIS2_HOME":                      {Priority: 11, Name: "DHIS2 Home", DefaultValue: &dhis2CoreDefaults.dhis2Home},
```
Comment (tonsV2) [inline pkg/stack/stack.go:256]: Same as above?

**Review by tonsV2 — APPROVED** [review]

File: pkg/stack/stack.go (hunk 3 above)
Comment (radnov) [reply pkg/stack/stack.go:187]: Changed to "DHIS2 Home Directory"

**Pull request merged**